ddtrace/_monkey.py

```diff
@@ -145,8 +145,8 @@ class ModuleNotFoundException(PatchException):
     pass
 
 
-def _on_import_factory(module, prefix="ddtrace.contrib", raise_errors=True):
-    # type: (str, str, bool) -> Callable[[Any], None]
+def _on_import_factory(module, prefix="ddtrace.contrib", raise_errors=True, should_patch=True):
+    # type: (str, str, bool, bool) -> Callable[[Any], None]
```

Comment:
is this supposed to be `Union[List[str], bool]` ?

or for this flow it can only be a bool?

Reply:
I wonder if we should change the name of `should_patch` to something else, since that only applies if it is a list of strings... ?


unless we add a `if not should_patch: return` ? (I know we filter out modules that shouldn't be patched, but just wondering since it could _technically_ be False based on the typing..... wdyt?)

Reply:
Good point. Does `patch_indicator` sounds better?